mkdocs.yml

```diff
@@ -1,20 +1,52 @@
 site_name: MAD at Freeletics
-site_url: https://freeletics.github.io/freeletics-mad/
+site_url: https://freeletics.github.io/mad/
 repo_name: MAD Freeletics
-repo_url: https://github.com/freeletics/freeletics-mad
+repo_url: https://github.com/freeletics/mad
 site_description: "Modern Android Development the Freeletics way"
 site_author: Freeletics
-remote_branch: gh-pages
 
 copyright: 'Copyright &copy; 2021 Freeletics.'
 
 theme:
   name: 'material'
-  favicon: images/icon-freeletics.png
-  logo: images/icon-freeletics.png
-  palette:
-    primary: 'black'
-    accent: 'white'
+  favicon: images/icon-freeletics.svg
+  logo: images/logo-freeletics.svg
+  features:
+    - navigation.sections
```

Comment:
Will make the side bar flat instead of having drop downs
<img width="224" alt="Screenshot 2023-01-13 at 13 20 47" src="https://user-images.githubusercontent.com/1358105/212319084-0e1f1d5d-b55f-4f00-bf5c-58460301f6c9.png">